### DISCUSSION TITLE
How to connect security onion to another elasticsearch ?

### DISCUSSION THREAD

**Discussion by ngocnn97** [Security-Onion-Solutions/securityonion#8465] [Unsupported Versions] [answered]
Hello !

I have successfully installed Security Onion 2.3.130 standalone and it works perfectly
The problem here is that my company also has another elasticsearch system and I want to connect my security onion to this system (I will not use elasticsearch on standalone anymore and will use it on other elasticsearch system)

Can anyone help me?

**Comment by dougburks** [comment, marked as answer]
You may be able to use Elastic cross cluster search to query to or from an external Elasticsearch instance. However, we don't provide any support for external Elasticsearch instances.

**Reply by ngocnn97** [reply]
> You may be able to use Elastic cross cluster search to query to or from an external Elasticsearch instance. However, we don't provide any support for external Elasticsearch instances.

Thanks for the help
